pkg/u-boot/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN eve-alpine-deploy.sh
 
 SHELL ["/bin/bash", "-eo", "pipefail", "-c"]
 
-ENV VERSION v2021.10
```

Comment:
Should we remove patches for `v2021.10` as well? I think we can find them in git tree if needed.

Reply:
@giggsoff I also prefer to leave one revision of the old patches, for greater clarity, that when updating, no patches were lost, it's easier than looking at git history.